### ISSUE TITLE
sigsegv in mattermost.SendWebhook

### ISSUE THREAD

**Issue by pgeorgi** [42wim/matterbridge#429]
**Describe the bug**
I run matterbridge to connect an IRC channel and a mattermost channel. mattermost -> IRC works, but each message on IRC crashes matterbridge, so the message doesn't arrive and the bridge rejoins.

The output is (using the docker container 1.10.0):
```
time="2018-05-19T05:31:15Z" level=info msg="Running version 1.10.0 86865c6" prefix=main
time="2018-05-19T05:31:15Z" level=info msg="Parsing gateway default" prefix=gateway
time="2018-05-19T05:31:15Z" level=info msg="Starting bridge: irc.freenode " prefix=gateway
time="2018-05-19T05:31:15Z" level=info msg="Connecting irc.freenode.net:6667" prefix=irc
time="2018-05-19T05:31:16Z" level=info msg="Connection succeeded" prefix=irc
time="2018-05-19T05:31:16Z" level=info msg="irc.freenode: joining #coreboot (ID: #corebootirc.freenode)" prefix=bridge
time="2018-05-19T05:31:16Z" level=info msg="Starting bridge: mattermost.site " prefix=gateway
time="2018-05-19T05:31:16Z" level=info msg="Connecting using webhookurl (sending) and webhookbindaddress (receiving)" prefix=mattermost
time="2018-05-19T05:31:16Z" level=info msg="mattermost.site: joining irc (ID: ircmattermost.site)" prefix=bridge
2018/05/19 05:31:16 Listening on http://0.0.0.0:9999...
time="2018-05-19T05:31:16Z" level=info msg="Gateway(s) started succesfully. Now relaying messages" prefix=main
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0x55eda293c0c2]

goroutine 163 [running]:
github.com/42wim/matterbridge/bridge/mattermost.(*Bmattermost).sendWebhook(0xc42041a8a0, 0xc420196100, 0x3f, 0xc420400d9c, 0x3, 0xc420586f00, 0xb, 0xc420457dc0, 0x1c, 0xc420196a80, ...)
        /go/src/github.com/42wim/matterbridge/bridge/mattermost/mattermost.go:388 +0x662
github.com/42wim/matterbridge/bridge/mattermost.(*Bmattermost).Send(0xc42041a8a0, 0xc420196100, 0x3f, 0xc420400d9c, 0x3, 0xc420586f00, 0xb, 0xc420457dc0, 0x1c, 0xc420196a80, ...)
        /go/src/github.com/42wim/matterbridge/bridge/mattermost/mattermost.go:132 +0x70d
github.com/42wim/matterbridge/gateway.(*Gateway).handleMessage(0xc4203f1810, 0xc420196100, 0x3f, 0xc420400d9c, 0x3, 0xc420586f00, 0xb, 0xc420457dc0, 0x1c, 0xc420196a80, ...)
        /go/src/github.com/42wim/matterbridge/gateway/gateway.go:275 +0x475
github.com/42wim/matterbridge/gateway.(*Router).handleReceive(0xc4204152e0)
        /go/src/github.com/42wim/matterbridge/gateway/router.go:101 +0x2ce
created by github.com/42wim/matterbridge/gateway.(*Router).Start
        /go/src/github.com/42wim/matterbridge/gateway/router.go:57 +0x579
```

**Expected behavior**
messages passing back and forth, no segfaults.

**Environment (please complete the following information):**
 - OS: Linux
 - Matterbridge version: docker images 42wim/matterbridge:1.10.0 (but also latest)

**Additional context**
The mattermost-side config is:
```
[mattermost]
    [mattermost.site]
    Server="chat.coreboot.org"

    # use webhooks. URL is incoming, Bind is outgoing (from mattermost's POV)
    # the URLs look weird but are a-ok from within the containers' POV
    WebhookURL="http://XXX/hooks/XXX"
    WebhookBindAddress="0.0.0.0:9999"
    IconURL="https://www.coreboot.org/images/1/17/Coreboot64px.png"

    SkipTLSVerify=true
    NoTLS=false

    Team="coreboot"
    #Login="XXX"
    #Password="XXX"
    Token="XXX"
    #PrefixMessagesWithNick=true
    RemoteNickFormat="<{NICK}> "
```

It crashes at the same place with both login/password and token authentication

**Comment by 42wim** [comment]
As a workaround, you can remove the WebhookURL and WebhookBindAddress if you have Token or Login/Password specified. (unless you need specific webhook features)

**Comment by 42wim** [comment]
Thanks for reporting, it's fixed in master now, a 0.10.1 will be released soon
